public/locales/de/translation.json

```diff
@@ -165,6 +165,11 @@
     "switchToUserMenu": "Wechsle zum User-Menu",
     "switchToAdminMenu": "Wechsle zum Admin-Menu"
   },
+  "Hotkeys": {
+    "hotkeysEnabled": "Hotkeys aktiviert!",
```

Comment:
Hotkey oder Tastaturkuerzel?

Reply:
According to [Duden](https://www.duden.de/rechtschreibung/Hotkey), Hotkey is a proper German word so I would leave it at that 😄